roles/rojinebrahimi.wp_mysql/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+# WORDPRESS MYSQL DATABASE CREDENTIALS
+mysql_password: "Qw9M#sL!B"
+wp_db_name: "wordpressdb"
+wp_db_password: "password"
```

Comment:
some variables here are duplicate each other. Can you set something dummy here and overwrite when you execute your role?

Reply:
Actually, mysql_password is the root password of the service. and the other is the database's password. Should I merge them?